content/docs/reference/spec/migration/buildpack-api-0.7-0.8.md

```diff
@@ -0,0 +1,52 @@
++++
+title="Buildpack API 0.7 -> 0.8"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.8) for buildpack API 0.8 for the full list of changes and further details.
+
+### Process-Specific Working Directory
+
+Buildpacks may specify the working directory for a process by setting the `working-dir` field on the process in [`launch.toml`](https://github.com/buildpacks/spec/blob/buildpack/0.8/buildpack.md#launchtoml-toml).
+
+Prior to this, all processes ran from the app directory (`CNB_APP_DIR`).
```

Comment:
Make it clear that `CNB_APP_DIR` is a build time variable:
```suggestion
Prior to this, all processes ran from the app directory (configurable at image build time through `CNB_APP_DIR`).
```

Reply:
As I understand it, `CNB_APP_DIR` must be set to the app directory by the platform, as input to the lifecycle phases (builder, detector, exporter, launcher). I don't think it's configurable from the buildpack author / buildpack API perspective.

That said, perhaps `CNB_APP_DIR` here is not clarifying, and we could just omit the reference to it. What do you think?